lib/root-dir.js

```diff
@@ -1,8 +0,0 @@
-const {app, process: remoteProcess} = require('electron').remote
-const minimist = require('minimist')
-const fs = require('fs')
-
-const argv = minimist(remoteProcess.argv.slice(2))
-const rootDir = argv.data || `${app.getPath('downloads')}/dat`
```

Comment:
oh yeah sounds good - we should probably come up with a better way to handle options later, but re-adding :sparkles: